.github/workflows/mypy.yml

```diff
@@ -37,3 +38,17 @@ jobs:
           cache-dependency-path: Tools/clinic/requirements-dev.txt
       - run: pip install -r Tools/clinic/requirements-dev.txt
       - run: mypy --config-file Tools/clinic/mypy.ini
+
+  mypy-cases-generator:
```

Comment:
Nitpick: let's put these jobs in alphabetical order, so this one first.

Or does it make sense to combine them into a single job, in their own steps? With a shared requirements file?

Reply:
OTOH there's quite a bit of duplication between the two sections. If something has to change we'd have to change it in each section. I don't how the first thing about how GitHub workflow .yml files work, so maybe that's unavoidable, but (as the expert for cases_generator and also a past mypy maintainer) I'd be happy to fix things quickly when a mypy upgrade requires it, and I'm sure we could coordinate with the AC expert.

Reply:
Sure, I don't feel too strongly about it! We can merge the two sections and see how it goes. I think the AC expert here is me (at least when it comes to the type annotations in `Tools/clinic/`), so we'd just have to talk to each other :-)

If we want to have a common `requirements-dev.txt` file shared between the two projects, where should it live? Just in `Tools/`?

Reply:
I will create a common directory as Tools/.mypy it will be fine?

Reply:
Not a fan of `.` files everywhere (too easy to miss with `ls`).

Reply:
I agree with Guido; `.` and `_` prefixed filenames are often hidden by various tools, which is irritating.